.github/workflows/release.yml

```diff
@@ -10,14 +10,16 @@ env:
 
 jobs:
   macos:
-    runs-on: macos-latest
+    runs-on: macos-11.0
 
     steps:
       - uses: actions/checkout@v2
+      - name: Install targets
+        run: rustup update && rustup target add x86_64-apple-darwin aarch64-apple-darwin
```

Comment:
I added it to be explicit that we are requiring both. Since this exact command can be run locally on a macOS arm64 machine to install the required targets. 

I'll add this exact invocation to install.md